src/features/artifacts/stories/ArtifactsList.stories.tsx

```diff
@@ -4,19 +4,19 @@ import { ArtifactList } from "../components";
 
 const artifactList = [
   {
-    name: "Link to lockfile",
+    name: "Show lockfile",
     route: "/api/v1/build/1/lockfile/"
   },
   {
-    name: "Link to yml file",
+    name: "Show yml file",
     route: "/api/v1/build/1/yaml/"
   },
   {
-    name: "Link to archive",
+    name: "Download archive",
```

Comment:
Is the default behaviour to download the archive?
i.e. if I click here will it automatically download artifacts?

Reply:
Ah nevermind - yes this is the default.

It now feels weird to have all the `Show` then a `Download` then a `Show`, can you move this `Download archive` after `Show conda... log`? That will make the whole thing more consistent.
Once that small fix is in I can merge right away

Reply:
Ok!